qt/scientific_interfaces/Indirect/IndirectDataAnalysis.ui

```diff
@@ -23,7 +23,7 @@
    </size>
   </property>
   <property name="windowTitle">
-   <string>Indirect Data Analysis</string>
+   <string>Inelastic Data Analysis</string>
```

Comment:
Would it be worth renaming the file here as well?

Reply:
From the discussion with @AnthonyLim23, there is a complete overhaul needed. I have raised #31508 accordingly. I changed here as it is the GUI window title, and the scientists/users who normally use `Direct ` related tools could get confused.